core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/AwsKeySigningMetadata.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core.multikey.metadata;
+
+import tech.pegasys.web3signer.core.config.AwsAuthenticationMode;
+import tech.pegasys.web3signer.core.config.AwsSecretsManagerParameters;
+import tech.pegasys.web3signer.core.signing.ArtifactSigner;
+import tech.pegasys.web3signer.core.signing.KeyType;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AwsKeySigningMetadata extends SigningMetadata implements AwsSecretsManagerParameters {
+
+  private final AwsAuthenticationMode authenticationMode;
+  private final String region;
+  private final String accessKeyId;
+  private final String secretAccessKey;
+  private final String secretName;
+
+  @JsonCreator
+  public AwsKeySigningMetadata(
```

Comment:
Adding a file parsing test would be good as well then we would know jackson is all setup correctly, we did this for interlock and yubihsm see e.g. InterlockSigningMetadataFileParsingTest